riscv/src/runtime64.rs

```diff
@@ -0,0 +1,662 @@
+use std::{collections::BTreeMap, convert::TryFrom};
+
+use powdr_riscv_syscalls::Syscall;
+
+use itertools::Itertools;
+
+use crate::code_gen::Register;
+
```

Comment:
This file contains everything that was "removed" from file above `runtime.rs`, with all the 64-bit specific stuff, unmodified.